packages/p2p/src/utils/format-value.js

```diff
@@ -0,0 +1,13 @@
+export const roundOffDecimal = (number, decimal_place = 2) => {
+    // Rounds of the digit to the specified decimal place
+    return parseFloat(Math.round(number * Math.pow(10, decimal_place)) / Math.pow(10, decimal_place));
+};
+
+export const setDecimalPlaces = (value, expected_decimal_place) => {
```

Comment:
NIT: Since the function name is `setDecimalPlaces`, `expected_decimal_places` would be a more accurate variable name than `expected_decimal_place` imo.

Reply:
Will rename the variable